app-web/src/hoc/Layout.js

```diff
@@ -27,6 +28,9 @@ export const Layout = ({ children }) => {
 
   return (
     <StyledContainer fluid>
+      <Helmet>
+        <script src="https://cdn.polyfill.io/v2/polyfill.min.js" />
```

Comment:
lets add a comment here explaining this is to fix some issues with running in ie 11

Reply:
done!